sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -217,6 +217,18 @@ object ScalaJSPlugin extends AutoPlugin {
         "`scalaJSUseMainModuleInitializer` is true",
         CTask)
 
+    val scalaJSLinkerConfig = SettingKey[StandardLinker.Config](
+        "scalaJSLinkerConfig",
+        "Configuration of the Scala.js linker",
+        BPlusSetting)
+
+    val scalaJSLinkerBatchMode = SettingKey[Boolean](
+        "scalaJSLinkerBatchMode",
+        "Whether to run the Scala.js linker in batch mode, where nothing is " +
+        "cached nor incremental. This reduces the memory footprint, which " +
+        "can be particularly useful in a CI environment.",
+        BMinusSetting)
```

Comment:
I'm not sure I buy this last argument. *If* it is ok to set `scalaJSLinkerBatchMode in Global` it is also OK to set `scalaJSLinkerConfig in Global` (I'm unsure *whether* it is, but that's orthogonal). Therefore one can just modify the global default config.

IMO being able to write `set scalaJSLinkerBatchMode in Global := true` instead of `set scalaJSLinkerConfig in Global ~= (_.withBatchMode(true))` or even `set every scalaJSLinkerConfig ~= (_.withBatchMode(true))` does not outweigh the benefits of having a single unified config that can span all build systems).